.github/workflows/ci.yml

```diff
@@ -1,4 +1,5 @@
 name: CI
+
 on:
   # pull_request: # we use Bors
```

Comment:
@DilumAluthge as far as I can tell this comment is false?

Reply:
Yeah, we now use GitHub Merge Queue instead. (Bors shut down their free public Bors instance after GitHub Merge Queue exited beta.)

Reply:
Do you? There's no CI at all on PRs, that line is commented out.

Reply:
Ah, yeah it looks like I never made the transition.

https://github.com/JuliaLang/BumpStdlibs.jl/pull/110